src/main/resources/com/adobe/epubcheck/schema/30/package-30.sch

```diff
@@ -295,5 +295,62 @@
     </pattern>
     
     <include href="./mod/id-unique.sch"/>
-
-</schema>
+	
+	<!-- EPUB 3.2 New Checks -->
```

Comment:
These changes look good!
However before merging we should test all these new warnings, each in its own test case…